cohorts/load.py

```diff
@@ -299,7 +317,7 @@ def plot_survival(self, on, col=None, col_equals=None, how="os", threshold=None)
         results = plot_kmf(
             df=df,
             condition_col=plot_col,
-            censor_col=self.dead_col if how == "os" else self.progressed_col,
+            censor_col=self.dead_col if how == "os" else self.progressed_or_dead_col,
```

Comment:
I meant if someone did define an OS censor columns and PFS censor column, it seems like a good idea to alert when they are out of sync, but still use each censor column. Basically, I can't think of a case of why they'd be out of sync when it wasn't an error, in that case it seems worthwhile to explicitly fix the data? And if there ever was a use-case for that when it wasn't an error, this would override that